developer/src/kmcmplib/tests/gtest-kmx_u16-test.cpp

```diff
@@ -0,0 +1,60 @@
+#include <gtest/gtest.h>
+#include "../src/kmx_u16.h"
+
+/*class kmx_u16_Test : public testing::Test {
+    protected:
+    	kmx_u16_Test() {}
+	    ~kmx_u16_Test() override {}
+	    void SetUp() override {}
+	    void TearDown() override {}
+};*/
```

Comment:
Are these stubs needed?

Reply:
Not really, but we may need them again soon as we extend the tests.